product_docs/docs/biganimal/release/overview/02_high_availability.mdx

```diff
@@ -4,25 +4,51 @@ redirects:
 
 By default, replication is synchronous to one replica and asynchronous to the other. That is, one replica must confirm that a transaction record was written to disk before the client receives acknowledgment of a successful commit. In PostgreSQL terms, `synchronous_commit` is set to `on` and `synchronous_standby_names` is set to `ANY 1 (replica-1, replica-2)`. You can modify this behavior on a per-transaction, per-session, per-user, or per-database basis with appropriate `SET` or `ALTER` commands.
 
-## High availability—not enabled
+## Extreme high availability (beta)
 
-For nonproduction use cases where high availability is not a primary concern, a cluster deployment with high availability not enabled provides one primary with no standby servers for failover or read-only workloads. 
+Extreme high availability clusters are powered by EDB Postgres Distributed, a logical replication tool that delivers more advanced cluster management compared to a physical replication based system. The primary component of EDB Postgres Distributed is Bi-Directional Replication (BDR). BDR is a PostgreSQL extension enabling higher performance and faster failover recovery compared to standard high availability.
+
+Extreme high availability clusters can be deployed with either EDB Postgres Advanced Server or EDB Postgres Extended Server as the database type. EDB Postgres Extended Server is PostgreSQL compatible. 
+
+The components of an extreme high availability cluster are all in one cloud region with the nodes distributed among three availability zones (A, B, C). One BDR node is designated the *leader* node (A.1). High Availability Routing for Postgres (HARP) routes all application traffic in a single data center or region to the leader node, which acts as the principle write target to reduce the potential for data conflicts. HARP leverages a distributed consensus model to determine availability of the BDR nodes in the cluster. On failure or unavailability of the leader, HARP elects a new leader and redirects application traffic. Together with the core capabilities of BDR, this mechanism of routing application traffic to the leader node enables fast failover and switchover without risk of data loss. 
+
+The leader node exists in one availability zone along with a *shadow* node. In another availability zone, there are two more shadow nodes. A third availability zone contains a *witness* node. The shadow nodes wait to take over when needed. If the leader node loses contact, BigAnimal switches immediately to a shadow node to continue processing. The witness node, as an odd numbered node, can make decisions that require consensus, such as generating new global sequences or distributed DDL operations. This is a normal BDR node that is deliberately set up not to replicate any tables or data to it.
```

Comment:
```suggestion
Extreme High Availability clusters deploy four data-hosting "BDR" nodes across two availability zones. One of these nodes will be the leader at any given time. The rest are typically referred to as "shadow" nodes. Any shadow node can be promoted to leadership at any time by the HARP router. The third availability zone contains one node called the "witness". This node does not host data; it exists only for management purposes, to support operations that require consensus in case of an availability zone failure.
```